src/control/nav2_dwa_controller/include/nav2_dwa_controller/dwa_controller.hpp

```diff
@@ -25,7 +25,7 @@ class DwaController : public nav2_tasks::FollowPathTaskServer
 {
 public:
   explicit DwaController(const std::string & name);
-  DwaController() = delete;
+  DwaController();
```

Comment:
Why is the constructor with a name explicit, but not the constructor without a name?